Admin updates in South Africa

stepps00 opened the pull request:
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1991.

This PR will:
- [x] Update county placetype to reflect new names for district municipalities
- [x] Add polygon geometries to localadmin records
- [x] Supersede/merge localadmin records, as needed
- [x] Review locality geometries and coverage, clipping to parent geometries and updating names when needed
- [x] Add/update coterminous properties as needed
- [x] PIP all updated records + descendants
- [x] Validate all json and geometries

stepps00 commented:
Commit Notes:
- [88278fa](https://github.com/whosonfirst-data/whosonfirst-data-admin-za/pull/14/commits/88278fa44aaa741fcbc85758c6de28bc8062fcbe) and [6462a33](https://github.com/whosonfirst-data/whosonfirst-data-admin-za/pull/14/commits/6462a33d1ec1b3a5143e4a584027550f774164aa): Update the county placetype in South Africa, superseding/new records/flagging as is_current = 0

nvkelso commented on data/110/873/050/9/1108730509.geojson:
```diff
@@ -11,11 +11,14 @@
     "iso:country":"ZA",
     "label:eng_x_preferred_longname":[
-        "Johannesburg Municipality"
+        "City of Johannesburg Metropolitan Municipality"
```
The labels are meant to be short and sweet. This sounds like a legal name, not a display name. Should the new form go into the name space instead?

stepps00 replied:
I've updated this in a recent commit.. preferring "City of Johannesburg" as the preferred label, moving the full string to a variant.

stepps00 commented:
Source: https://github.com/whosonfirst/whosonfirst-sources/pull/193

stepps00 commented:
Merging this PR shortly. Geometries, properties, and json have been validated. The data source has also been merged.

Summary of changes:
- Updated county geometries
- New localadmin Polygon geometries, superseding work at this placetype, too
- Cleaned up Polygon geometries around four big cities in South Africa
- Cleaned up neighbourhoods and locality points in/around the same big cities
- PIP work to update all hierarchies/parent_ids
